src/client.js

```diff
@@ -37,6 +37,9 @@ if (ENV !== envs.TESTING) {
 }
 
 // Other client listeners
-client.on('message', (message) => dispatchCommand(message));
+client.on(
+  'message',
+  (message) => ENV !== envs.PRODUCTION && dispatchCommand(message)
```

Comment:
What's the reason for this?

Reply:
feature flag, but simple version without using launch darkly or rollout apis